en/01_Commerce/v1/50_Modules/Discounts/UserDiscounts.md

```diff
@@ -12,4 +12,4 @@ User and user group discounts can be restricted to:
 
 When a user(group) discount is available to the logged in user, it is automatically applied to the items in their cart. 
 
-If multiple user(group) discounts are valid, they will all be applied. 
\ No newline at end of file
+If multiple user(group) discounts are valid, they will all be applied. 
```

Comment:
You have to change this here too: https://github.com/modmore/docs/search?q=UserDIscounts